apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Logging/GoogleLogger.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright 2016 Google Inc. All Rights Reserved.
+﻿// Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
Just an FYI you don't need to update these dates each time.  You're welcome to but not required unless it's a new file.

Reply:
Understood 👍